### DISCUSSION TITLE
Shelly Plus 2 PM as detached button

### DISCUSSION THREAD

**Discussion by LeeWiu** [arendst/Tasmota#20605] [Q&A] [answered]
Hi, I used without issues a Shelly 2.5 as a detached button with default template and added in console:
`Backlog SetOption1 1; SetOption11 1; SetOption32 20; SetOption73 1`
And the rule
`rule1 on button1#state=10 do power1 toggle endon on button2#state=10 do power2 toggle endon
`
Now I converted a Shelly Plus 2 PM to Tasmota v13.3.0(solo1) but I can't setup as a button.
I want to use action double, triple, hold...
Can someone help me please with a working template and commands?
Shelly 2.5
<img width="353" alt="Screenshot 2024-01-27 at 12 05 00" src="https://github.com/arendst/Tasmota/assets/1405188/3866e2d6-5d4b-43a6-ae57-3b64260d547a">
Shelly Plus 2PM
<img width="344" alt="Screenshot 2024-01-27 at 13 14 58" src="https://github.com/arendst/Tasmota/assets/1405188/7cc655de-aee5-48b3-a756-d43287983417">

**Comment by barbudor** [comment, marked as answer]
Why not doing the same change on the Shelly Plus 2 PM as you already done on the Shelly 2.5

You already replaced Switch_n by Button_in on the Shelly 2.5 so it again on the Shelly Plus 2PM
If Button_in (inverted) doesn't work as expected, use Button_n)

**Comment by LeeWiu** [comment]
Thank you for heads up, I did it with template:
`{"NAME":"Shelly Plus 2PM PCB v0.1.9","GPIO":[320,0,0,0,0,128,0,0,225,224,0,0,0,0,129,0,0,0,0,0,0,608,640,3458,0,0,0,0,0,9472,0,4736,0,0,0,0],"FLAG":0,"BASE":1}`

<img width="338" alt="Screenshot 2024-01-27 at 16 00 36" src="https://github.com/arendst/Tasmota/assets/1405188/8294932e-d67e-4ee2-952f-339ffe899ee3">
